tracer/test/test-applications/integrations/Samples.Wcf/Program.cs

```diff
@@ -26,26 +26,26 @@ private static async Task Main(string[] args)
             {
                 binding = new WSHttpBinding();
                 baseAddress = new Uri($"http://localhost:{port}/{WcfNamespace}/");
-                expectedExceptionCount = 0;
+                expectedExceptionCount = 2;
             }
             else if (args.Length > 0 && args[0].Equals("BasicHttpBinding", StringComparison.OrdinalIgnoreCase))
             {
                 binding = new BasicHttpBinding();
                 baseAddress = new Uri($"http://localhost:{port}/{WcfNamespace}/");
-                expectedExceptionCount = 0;
+                expectedExceptionCount = 2;
             }
             else if (args.Length > 0 && args[0].Equals("NetTcpBinding", StringComparison.OrdinalIgnoreCase))
             {
                 binding = new NetTcpBinding();
                 ((NetTcpBinding)binding).TransferMode = TransferMode.Streamed;
                 baseAddress = new Uri($"net.tcp://localhost:{port}/{WcfNamespace}/");
-                expectedExceptionCount = 0;
+                expectedExceptionCount = 2;
             }
             else if (args.Length > 0 && args[0].Equals("Custom", StringComparison.OrdinalIgnoreCase))
             {
                 binding = ConfigureCustomBinding();
                 baseAddress = new Uri($"http://localhost:{port}/{WcfNamespace}/");
-                expectedExceptionCount = 7;
+                expectedExceptionCount = 13;
```

Comment:
Quick question: Do you know why this amount increases to 13? It's been so long since I looked at this sample that I don't remember at the top of my head

Reply:
The CustomBinding test throws for every request. And now there are 6 extra requests 🙂